src/main/java/servlets/LoginServlet.java

```diff
@@ -36,11 +37,8 @@
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     response.setContentType(CONTENT_TYPE_TEXT_HTML);
-    PrintWriter out = response.getWriter();
-    if (UserAuthUtil.isUserLoggedIn()) {
-      out.println(UserAuthUtil.getLogoutURL(REDIRECT_LINK));
-    } else {
-      // Send user to OAuth consent page.
+    response.getWriter().println(ResponseBuilder.toJson(UserAuthUtil.isUserLoggedIn(), REDIRECT_LINK));
```

Comment:
Are you eventually going to redirect the user to the login/logout page? Correct me if I'm wrong, but it looks like you just print out the filepaths to get to the login/logout page instead of actually redirecting them to that page.

Reply:
#24

Reply:
see issue #24